core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -1128,3 +1134,51 @@ def __create_boot_dot_properties(self):
             ostream.close()
         self.logger.exiting(class_name=self.__class_name, method_name=_method_name)
         return
+
+    def __configure_opss_secrets(self):
```

Comment:
Dont we need to screen off the online use case?  You cannot update the secret in online mode, can you?

Reply:
It's for offline , not online

Reply:
and during creation.

Reply:
i.e. it's not even in the code path for online